import-service/src/functions/import-file-parser/handler.ts

```diff
@@ -0,0 +1,46 @@
+import 'source-map-support/register';
+
+import { middyfy } from '@libs/lambda';
+
+import type { Handler, S3Event } from 'aws-lambda';
+import AWS from 'aws-sdk';
+import csv from 'csv-parser';
+
+const importFileParser: Handler<S3Event, void> = async (event) => {
+    const bucket = process.env.S3_BUCKET;
+    const s3 = new AWS.S3({ region: 'eu-west-1' });
+
+    console.log('Parse import file request', event);
+
+    try {
+        for (const record of event.Records) {
```

Comment:
I think it does not make sense to loop over the `Records`, since with the following solution there should be always 1 Record with `ObjectCreated` event data

Reply:
That's true, but I think that the general approach with the for loop is always better - if, for whatever reason. the is more than one record at some point in the future, this code won't have to be refactored